### ISSUE TITLE
did a fresh install of "dev 2.7.0 2018-11-04" and discovered these issues

### ISSUE THREAD

**Issue by tringate** [Program-O/Program-O#470]

**Comment by Dave-Morton** [comment]
There's an extra backtick operator (`) in the SQL query, so that query isn't going to fly. Not sure where the actual bug is, but it would be good to look near line number 293 in select_bots.php for the declaration of the SQL query. This is most likely where you can find the source of the issue.

BTW, @tringate , I have Skype reinstalled on my main machine. Please contact me when you have time. I'd like to catch up.

**Comment by samiam123** [comment]
correct, its because unique was set in the migration to sql in that table.
The main point of his test was to pass the logic to mysql with NO DUP
It will freek on it updating a unique coulomb.
PhP and MY sq will not update a table that is unique. so it throws a fit as it should.
The main test was to see if program-o can migrate on update new structure with out client doing nothing more than an update to end user.
Your right, that table in that search or update should not be "Unique".
Now what?

**Comment by samiam123** [comment]
What is and needs to happen is what your seeing.
1. many of you have well made bots.
2. You want new powers as botmaster
3. Mabey you dont care and want a new bot.
4. Ask how do we take all of your work and add in extra?
5. assuming 4 , how do we NOT break anything existing.
Migration is what we do.
We dont break any data or logic you have now but add or correct a new table.
thats what migration is.
But yes there are other errors in the lib and need fixed too. (php changed alot and mysql too).
Anyway other than it filling up you logs, it still works.

**Comment by samiam123** [comment]
migration pains, cant add to unique locked. Minor correction to the migration is all.
also PDO lib changed
Migration is whats needed and thats all growing pains.
Unique LOCKS update on a change so it bitches.
So, How are you now? lol
